Tech-skill Add section layout and styles

Vitaly-Stefanyshyn777 opened the pull request.

Vitaly-Stefanyshyn777 committed:
Add section layout and styles

yuliia-shl commented on src/css/tech-skills.css:
```diff
@@ -0,0 +1,182 @@
+/* Основні стилі секції Tech Skills */
+.tech-skills-section {
+  position: relative;
+  padding: 32px 0;
```
треба замінити чи видалити, бо нижній і верхній паддінги для секції прописані в глобальних стилях

yuliia-shl commented on src/partials/tech-skills.html:
```diff
@@ -0,0 +1,85 @@
+      <ul class="tech-marquee tech-marquee-top">
+        <li class="tech-marquee-item">
+          <span class="tech-skill-text">HTML/CSS</span>
+          <svg class="tech-icon" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
усі svg треба винести в спрайт і підключити за посиланням           

![Screenshot 2024-10-10 172526](https://github.com/user-attachments/assets/fe65772d-e79a-4e81-a3f2-efd9e894986c)

yuliia-shl commented on src/partials/tech-skills.html:
```diff
@@ -0,0 +1,85 @@
+    <div class="tech-skills-content">
+      <ul class="tech-marquee tech-marquee-top">
+        <li class="tech-marquee-item">
+          <span class="tech-skill-text">HTML/CSS</span>
```
Порядок тех-скілів не відповідає макету
![Screenshot 2024-10-10 173234](https://github.com/user-attachments/assets/22d384fb-7cbf-45ee-8145-8800bbc8a8e1)

yuliia-shl commented on src/partials/tech-skills.html:
```diff
@@ -0,0 +1,85 @@
+          <svg class="tech-icon" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <rect width="8" height="8" rx="2" transform="matrix(-1 0 0 1 16 8)" fill="#14C57C" />
+          </svg>
+          <span class="tech-skill-text">JavaScript</span>
```
Пропущені квадратики, колір на нижній лінії має бути чорний
![555](https://github.com/user-attachments/assets/e19c96a6-4efd-4f4c-a75c-ec37db428f90)

yuliia-shl commented on src/css/tech-skills.css:
```diff
@@ -0,0 +1,182 @@
+  font-weight: 500;
+  font-size: 36px;
+  line-height: 1.33333;
+  letter-spacing: 0.01em;
```
стилі для моб інші ніж для таб і деск
![Screenshot 2024-10-10 173729](https://github.com/user-attachments/assets/e3354729-5960-41f4-ac07-1653196be419)

yuliia-shl commented on src/css/tech-skills.css:
```diff
@@ -0,0 +1,182 @@
+.tech-skill-text {
+  font-weight: 500;
+  font-size: 36px;
+  line-height: 1.33333;
```
залишаємо тільки 2 знаки після крапки

yuliia-shl reviewed (changes requested):
підправ трохи, згідно коментів

Vitaly-Stefanyshyn777 committed:
Зберіг зміни в tech-skills

Vitaly-Stefanyshyn777 committed:
Merge branch 'main' into tech-skills

Vitaly-Stefanyshyn777 committed:
tech-skills-2

Vitaly-Stefanyshyn777 committed:
tech-skills-3

Vitaly-Stefanyshyn777 committed:
tech-skills-4

Vitaly-Stefanyshyn777 committed:
tech-skills-5

Vitaly-Stefanyshyn777 committed:
Merge remote-tracking branch 'origin/main' into tech-skills

Vitaly-Stefanyshyn777 committed:
tech-skills-6

white and black are already ready, desktop, tablet, mobile are also ready, you need to solve the problem with the font

Vitaly-Stefanyshyn777 committed:
tech-skills

я наріша твої макетні значення
